Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -543,6 +542,10 @@ export function extend(publicAPI, model, initialValues = {}) {
   if (typeof InstallTrigger !== 'undefined') {
     model.defaultToWebgl2 = true;
   }
+  model.defaultToWebgl2 = true;
```

Comment:
@martinken should I remove `defaultToWebgl2` altogether? So we always use webgl2 if available?

Reply:
I would leave it for now and see how it goes. After a while we can yank it if we find no cases where we need to fall back to webgl 1 even when webgl2 is available.